sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2-beta/async_samples/sample_analyze_general_documents_async.py

```diff
@@ -88,6 +88,11 @@ async def analyze_general_documents():
     print("----Entities found in document----")
     for entity in result.entities:
         print("Entity of category '{}' with sub-category '{}'".format(entity.category, entity.sub_category))
+        # NOTE: Calling get_lines() on each entity will return a list of DocumentLines that can be processed 
+        # just like any other DocumentLine instance.
+        line_length = len(entity.get_lines())
+        if line_length > 0:
+            print("...has {} line(s)".format(line_length))
```

Comment:
can we simplify to this like we do in other spots?

```suggestion
        print("...has {} line(s)".format(len(entity.get_lines())))
```

Reply:
So I was noticing that several entities have 0 lines...wasnt sure if it was worth it to print 0 lines for them. Let me know if you think it's still better to change this.

Reply:
interesting, do the entities have words?

Reply:
Looking into this, I was stepping through with the debugger to see if we got words when lines were 0 and this might be a bug...investigating.

Reply:
So it's not a bug per se, the spans for what is returned as a DocumentLine and a DocumentEntity do not technically match up. The lines are slightly longer than the recognized entity. This is an AI algorithm result, so for now we are doing the right thing, but we can talk to the service or consider improving our own algorithm to allow lines that "overlap" with the spans of our element. (This is probably why Paul was suggesting the mode option). And to answer your initial question yes the entities can have words that fall into their span.

Reply:
would it make sense to show get_words() on the entity here then? Since it seems more likely that an entity would be composed of some words than a full line, I think

Reply:
Yeah I can do that, I'm just looking for a good place to show off get_lines() which is why I added it here...maybe on key value pairs....but I think that may have the same problem.

Reply:
Did not end up adding that for entities as well, didnt seem necessary anymore. I did add another sample however that focuses on these new methods.